openedx/features/learner_profile/static/learner_profile/js/learner_profile_factory.js

```diff
@@ -103,6 +103,18 @@
             });
 
             sectionOneFieldViews = [
+                new FieldsView.DateFieldView({
+                    title: gettext('Joined'),
+                    titleVisible: true,
+                    model: accountSettingsModel,
+                    screenReaderTitle: gettext('Joined Date'),
+                    valueAttribute: 'date_joined',
+                    helpMessage: '',
+                    userLanguage: accountSettingsModel.get('language'),
+                    userTimezone: accountPreferencesModel.get('time_zone'),
+                    dateFormat: 'MMM YYYY'  // not localized, but hopefully ok.
```

Comment:
Yes, it does, according to the [Moment formatting guide](https://momentjs.com/docs/#/displaying/).